integration/api/api.py

```diff
@@ -104,6 +103,15 @@ def get_results(search_id):
     return Response(f"No search found with id {search_id}", status=404)
 
 
+@app.route("/api/system/about", methods=["GET"])
+def get_about():
+    global route_configs
+    conf = route_configs["about"]
+    if not conf["status_code"] == 200:
```

Comment:
This is neater: `if conf["status_code"] != 200`

Please change the other instances of this pattern in this file.